openstack/02b-MAChigh/neighbors.c

```diff
@@ -539,7 +540,9 @@ void neighbors_updateMyDAGrankAndNeighborPreference() {
             rankIncrease = DEFAULTLINKCOST*2*MINHOPRANKINCREASE;
          } else {
             //6TiSCH minimal draft using OF0 for rank computation
-            rankIncrease = (uint16_t)((((float)neighbors_vars.neighbors[i].numTx)/((float)neighbors_vars.neighbors[i].numTxACK))*2*MINHOPRANKINCREASE);
+            rankIncreaseIntermediary = (((uint32_t)neighbors_vars.neighbors[i].numTx) << 10);
```

Comment:
+1 in cutting the operations in several lines. This has bitten us a number of times in the past.